product_search_category_attribute_sale/wizards/product_search_category_sale_product_line.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductSearchCatSaleProductLine(models.TransientModel):
+    _name = "product.search.cat.sale.product.line"
+    _description = "Product Search Category Sale Product Line"
+
+    product_search_cat_attr_sale_id = fields.Many2one(
+        comodel_name="product.search.cat.attr.sale"
+    )
+    product_id = fields.Many2one(comodel_name="product.product", string="Product")
+    attribute_value_ids = fields.Many2many(
+        comodel_name="product.attribute.value",
+        compute="_compute_attribute_value_ids",
+        string="Attributes",
+    )
+    quantity = fields.Float()
+
+    @api.depends("product_id")
+    def _compute_attribute_value_ids(self):
+        for sel in self.filtered(lambda a: a.product_id.attribute_line_ids):
+            sel.attribute_value_ids = sel.product_id.attribute_line_ids.mapped(
```

Comment:
you should always give it a value, otherwise it will give strange odoo errors later on.

Reply:
I think there's no need from v16 or so, but I have changed it anyway.